### ISSUE TITLE
Recharge System!!!!!! 

### ISSUE THREAD

**Issue by UjjwalAtreya** [UjjwalAtreya/Chamena#1]
Lets make a extra table in database called Recharge.
And let User to Recharge his Canteen Code Number,like the user will visit the canteen and tells his unique canteen id and deposit some amount for his further ordering.

**Comment by sthasonal** [comment]
So we provide the user a card or is it only an account?

**Comment by UjjwalAtreya** [comment]
No card.just Canteen Code.

**Comment by UjjwalAtreya** [comment]
like if i have to recharge ill go to the canteen and give my Canteen Code Number and get some amount deposited in my Number.

**Comment by UjjwalAtreya** [comment]
www.youtube.com/watch?v=ahdVINa8wB0

**Comment by sthasonal** [comment]
ok so every user has an account and he deposits money in that account, but tyo paisa account ma jamma bhayo bhanera k user lai notify garne?

**Comment by UjjwalAtreya** [comment]
ah via mail bata..coz aaba phonne ma sms system will be biger so...lets make aakdam simple ..no monet costing activity..:D

**Comment by sthasonal** [comment]
mail bhanda ni usko canteen wala acount mai euta notification rakham na facebook notification jasto haina

**Comment by UjjwalAtreya** [comment]
ah tyo nai garna parcha notification ma mail bhanda...

**Comment by UjjwalAtreya** [comment]
what if a user didnt showed up order pick garna???tesle ta paisa tiri sakeko hunnhca???
@sthasonal

**Comment by sthasonal** [comment]
yes lets just send a notification like "You have deposited eti eti amounts in your account! Happy Eating! " yes good nice

**Comment by sthasonal** [comment]
tirisakeko huncha, uslai assign gareko time bhanda 15 minutes samma aipugena bhane notification pathaune account ma, your food is getting cold please come pick it up bhanera, tespachi ni aena bhane aena aba k garne

**Comment by sthasonal** [comment]
we have to deal with a lot of time wala codes esma myan
